Implement AsFd for Capture<Active>

gmacon opened the pull request:
I wanted to use [`nix::poll::poll`](https://docs.rs/nix/latest/nix/poll/fn.poll.html) to wait for the next packet, and a `BorrowedFd` is needed.

Wojtek242 commented:
Thanks for the PR! Looks good to me as it is. Can you just update the CHANGELOG as well? I will then release it shortly after I merge it.

gmacon commented:
Thank you for turning this around so fast. I had implemented equivalent functionality as a free function in anticipation of having to wait for this to end up in a release, but you got the release done before my colleagues got my changes reviewed, so I've just removed the free function and called this one instead. I really appreciate it.
